main.js

```diff
@@ -144,6 +145,11 @@ module.exports = function(options) {
 		next();
 	});
 
+	app.use(function(req, res, next) {
+		res.locals.jsonLd = [nextJsonLd.webSite()];
+		next();
+	});
```

Comment:
that sounds good

@GlynnPhillips, was thinking about APIs, things that don't output any using facing html but still use n-express